mmdt/polls/migrations/0002_question_is_enabled.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.4 on 2024-01-12 10:53
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('polls', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='question',
+            name='is_enabled',
+            field=models.BooleanField(default=True),
+        ),
+    ]
```

Comment:
If we update the database, should we also update the database diagram? The database diagram can be found in the readme.md file. You can generate the diagram using mysqllite. Please try to generate a new diagram with the updated database structure and notify me on Slack if you couldn't able to generate it.

[Nice to have]
It would be beneficial to mention in the pull request how to migrate this, so that other developers can follow the same steps when merging the pull request.

Reply:
- [ ] Hello @khinezarthwe let me know I should draw and update database diagram.